src/main/resources/synchronization/ru.yml

```diff
@@ -1,45 +1,53 @@
-#                                           в противном случае групповая синхронизация учитывает все группы, в которые входит игрок, включая родительские группы.
-# GroupRoleSynchronizationOnLink: повторная синхронизация при связывании игрока
-# GroupRoleSynchronizationCycleTime: число минут между запуском синхронизации всех игроков онлайн
-#
+###########################################################################
+# Синхронизация групп в Minecraft и ролей в Discord 
+# Требует установку плагина Vault!                  
+
+# Группы и роли, которые нужно синхронизировать
+# Используйте формат {"группа": "id роли в Discord"}
+# Чтобы узнать id роли, напишите "/discord debug" и посмотрите в первую секцию.
 GroupRoleSynchronizationGroupsAndRolesToSync: {"trusted": "000000000000000000", "vip": "000000000000000000"}
+
+# Если выдать/забрать группу в Minecraft, то соответствующая роль 
+# должна появиться/пропасть в Discord? true - да, false - нет
 GroupRoleSynchronizationMinecraftIsAuthoritative: true
+
+# Включены ли в Minecraft права discordsrv.sync.deny.<id роли>
 GroupRoleSynchronizationEnableDenyPermission: false
+
+# true - синхронизируется только главная группа игрока.
+# false - синхронизируются все побочные и родительские группы.
 GroupRoleSynchronizationPrimaryGroupOnly: false
+
+# Нужно ли проводить синхронизацию, когда игрок привязывает аккаунт в Discord?
 GroupRoleSynchronizationOnLink: true
+
+# Время (в минутах) - как часто DiscordSRV будет синхронизировать роли и группы игроков.
 GroupRoleSynchronizationCycleTime: 5
 
-# Синхронизация заблокированных пользователей
-# Если игрок получает бан на сервере и его запись связана с Discord, вы можете дополнительно забанить его и на сервере Discord и наоборот
-#
-# BanSynchronizationDiscordToMinecraft: банить ли игрока на сервере Minecraft, если он забанен на сервере Discord
-# BanSynchronizationDiscordToMinecraftReason: сообщение, которое будет использоваться для бана игроков на сервере Minecraft
-# BanSynchronizationMinecraftToDiscord: банить ли игрока на сервере Discord, если он забанен на сервере Minecraft
-#
+###########################################################################
+# Синхронизация банов 
+# Если игрока забанят в Minecraft, то он автоматически получит бан в дискорде (и наоборот)
+
+# Если игрока забанили в Discord, нужно ли банить его в Minecraft? true - да, false - нет
 BanSynchronizationDiscordToMinecraft: false
-BanSynchronizationDiscordToMinecraftReason: "&cВы забанены на нашем сервере, так как получили бан на нашем сервере Discord."
+# При бане через Discord, какая должна быть причина бана в Minecraft?
+BanSynchronizationDiscordToMinecraftReason: "&cВы были заблокированы на сервере Discord, поэтому ваш аккаунт в Minecraft тоже получил бан."
+# Если игрока забанили в Minecraft, нужно ли банить его в Discord? true - да, false - нет
 BanSynchronizationMinecraftToDiscord: false
+
```

Comment:
completely changed the formatting. Would like it to stay consistent with other translations for `synchronization.yml`.

Reply:
Restored the original structure of `synchronization.yml`